forch/faucet_state_collector.py

```diff
@@ -823,6 +823,23 @@ def process_port_learn(self, timestamp, name, port, mac, ip_addr):
                 devices_placement = DevicePlacement(switch=name, port=port, connected=True)
                 self._placement_callback(mac, devices_placement)
 
+    @_dump_states
+    # pylint: disable=too-many-arguments
```

Comment:
do you know why this is required but the one for the method below does not?   They both have 5 arguments, right?

Reply:
cleaned up.